```
error: builder for '/nix/store/x0kbm8437lf6fdagy22hrrbd5r8m34rz-python3.12-enlighten-1.14.0.drv' failed with exit code 1;
       last 25 log lines:
       >         self.tty.clear()
       >
       >         # Counter 1 in auto-refresh list
       >         self.assertIn(counter1, manager.autorefresh)
       >
       >         # If auto-refreshed counter hasn't been refreshed recently, refresh
       >         counter1.last_update = 0
       >         counter2.refresh()
       >         self.tty.stdout.write(u'X\n')
       >         output = self.tty.stdread.readline()
       >         self.assertRegex(output, 'counter2.+counter1')
       >
       >         # If auto-refreshed counter has been refreshed recently, skip
       >         counter1.last_update = counter1.start + 5.0
       >         counter2.refresh()
       >         self.tty.stdout.write(u'X\n')
       >         output = self.tty.stdread.readline()
       >         self.assertRegex(output, 'counter2')
       > >       self.assertNotRegex(output, 'counter1')
       > E       AssertionError: Regex matched: 'counter1' matches 'counter1' in '\x1b[6n\x1b[25;1H\r\x1b[Kcounter2\x1b[24;1H\r\x1b[Kcounter1\x1b[23;1HX\n'
       >
       > tests/test_manager.py:384: AssertionError
       > =========================== short test summary info ============================
       > FAILED tests/test_manager.py::TestManager::test_autorefresh - AssertionError: Regex matched: 'counter1' matches 'counter1' in '\x1b[6n\x1...
       > ============= 1 failed, 136 passed, 1 skipped in 92.83s (0:01:32) ==============
       For full logs, run 'nix log /nix/store/x0kbm8437lf6fdagy22hrrbd5r8m34rz-python3.12-enlighten-1.14.0.drv'.
error: 1 dependencies of derivation '/nix/store/57zzka2s27j8wqdq22kkffb566ciy235-python3.12-nsz-4.6.1.drv' failed to build
error: builder for '/nix/store/qx0fn0wzms2swka07hc6mzw18zv2qn9z-python3.13-enlighten-1.14.0.drv' failed with exit code 1;
       last 25 log lines:
       >         self.tty.clear()
       >
       >         # Counter 1 in auto-refresh list
       >         self.assertIn(counter1, manager.autorefresh)
       >
       >         # If auto-refreshed counter hasn't been refreshed recently, refresh
       >         counter1.last_update = 0
       >         counter2.refresh()
       >         self.tty.stdout.write(u'X\n')
       >         output = self.tty.stdread.readline()
       >         self.assertRegex(output, 'counter2.+counter1')
       >
       >         # If auto-refreshed counter has been refreshed recently, skip
       >         counter1.last_update = counter1.start + 5.0
       >         counter2.refresh()
       >         self.tty.stdout.write(u'X\n')
       >         output = self.tty.stdread.readline()
       >         self.assertRegex(output, 'counter2')
       > >       self.assertNotRegex(output, 'counter1')
       > E       AssertionError: Regex matched: 'counter1' matches 'counter1' in '\x1b[6n\x1b[25;1H\r\x1b[Kcounter2\x1b[24;1H\r\x1b[Kcounter1\x1b[23;1HX\n'
       >
       > tests/test_manager.py:384: AssertionError
       > =========================== short test summary info ============================
       > FAILED tests/test_manager.py::TestManager::test_autorefresh - AssertionError: Regex matched: 'counter1' matches 'counter1' in '\x1b[6n\x1...
       > ============= 1 failed, 136 passed, 1 skipped in 92.77s (0:01:32) ==============
       For full logs, run 'nix log /nix/store/qx0fn0wzms2swka07hc6mzw18zv2qn9z-python3.13-enlighten-1.14.0.drv'.
error: 1 dependencies of derivation '/nix/store/pyp44ngswkjgci8563m75p1fqs20vh3r-python3.13-nsz-4.6.1.drv' failed to build
error: 4 dependencies of derivation '/nix/store/zqmvmwpqygdj20s43q8k02pqw87zxg86-review-shell.drv' failed to build
```

--- a/pkgs/development/python-modules/enlighten/default.nix
+++ b/pkgs/development/python-modules/enlighten/default.nix
@@ -14,14 +14,14 @@
 
 buildPythonPackage rec {
   pname = "enlighten";
-  version = "1.14.0";
+  version = "1.14.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-qfpY4o5i3+9wg5D+6w7jxOZHgsACdGqUhzOQ/sF3/v4=";
+    hash = "sha256-hcNUEqmk84hrMzfUH4E0Qfq5ow2fW18MAVvQeKRBFHM=";
   };
 
   dependencies = [

